justfile

```diff
@@ -5,48 +5,74 @@ set dotenv-load
 PROJECT_ROOT_PATH := `DIR="${JUSTFILE:-justfile}"; DIR="$(realpath "$DIR")"; echo "$(dirname "$DIR")"`
 
 # In CI some image doesn't have scp and we can fallback to cp
-
 cp_cmd := `which scp || which cp`
 
-# Fork network and run tests for polkadot from the post-migration state
+# Run AHM for Polkadot
 default:
-    just run
+    just ahm polkadot
 
-# Run the network migration
-run:
-    just submodule-init
-    just submodule-update
-    just build-polkadot
+# ------------------------- ONE-TIME SETUP -------------------------
+
+# only to be run the first time, or when submodules changes.
+init:
+    git submodule update --init --recursive
+
+# only run once if any of the deps, like ZB, DG, or such are updated
+setup:
+    git submodule update --remote --merge
+    just install-doppelganger
+    just install-zombie-bite
     npm install
```

Comment:
I would put this inside the `ahm` command, as you might often change a TS file, which needs tsc recompilation, but doesn't need to re-install `zombie-bite` and `doppelganger`